src/securityPolicy.c

```diff
@@ -806,6 +874,7 @@ static inline security_policy_t _poolRegistrationOperatorWitnessPolicy(const bip
 
 	case PATH_ORDINARY_SPENDING_KEY:
 	case PATH_POOL_COLD_KEY:
+		// only ordinary spending key paths (because of inputs) and pool cold key path are allowed
 		WARN_UNLESS(bip44_isPathReasonable(path));
 		SHOW();
```

Comment:
Why to we need to show ordinary key paths here?

Reply:
I don't see any reason for that, but am wary of not showing them, I've just added a TODO.